release/release_gold.sh

```diff
@@ -110,23 +110,58 @@ function tag_release {
 		repository=liferay-portal
 	fi
 
-	if (! curl \
-			"https://api.github.com/repos/liferay/${repository}/git/tags" \
-			--data-raw '
-				{
-					"message": "",
-					"object": "'$(lc_get_property release-data/release.properties git.hash.liferay-portal-ee)'",
-					"tag": "'${LIFERAY_RELEASE_VERSION}'",
-					"type": "commit"
-				}' \
-			--fail \
-			--header "Accept: application/vnd.github+json" \
-			--header "Authorization: Bearer ${LIFERAY_RELEASE_GITHUB_PAT}" \
-			--header "X-GitHub-Api-Version: 2022-11-28" \
-			--max-time 10 \
-			--request POST \
-			--retry 3 \
-			--silent)
+	local username=liferay
+
+	if [ -n "${LIFERAY_RELEASE_USER_REPOSITORY}" ]
+	then
+		username=${LIFERAY_RELEASE_USER_REPOSITORY}
+	fi
+
```

Comment:
@lucasmiranda0, why did you need this? Can you please clarify?

Reply:
I was already thinking on the test perspective. If we don't have a variable like that, every time that we test we have to change the user in the code, otherwise we won't create a tag, unless you have permission to create a tag to the Liferay repository.